src/pint/models/binary_piecewise.py

```diff
@@ -0,0 +1,465 @@
+"""The BT (Blandford & Teukolsky) model.
```

Comment:
This should state that this is a piecewise version

Reply:
Corrected